README.adoc

```diff
@@ -163,76 +162,55 @@ http://json-b.net/users-guide.html[JSON Binding 1.0 Users Guide].
 
 == Consuming the REST resource
 
-Write a simple `Consumer` class that consumes the artist JSON using both JSON-B and JSON-P.
-
-`src/main/java/io/openliberty/guides/consumingrest/Consumer.java`:
+The `Artist` and `Album` POJOs are ready for deserialization. To consume the JSON response
+from your REST service create the `Consumer` class in the
```

Comment:
I think a comma is needed in "from your REST service, create the `Consumer` class in the"